Todo_TS/src/App.ts

```diff
@@ -0,0 +1,63 @@
+import TodoForm from './TodoForm';
+import TodoList from './TodoList';
+import TodoCount from './TodoCount';
+import TodoHeader from './TodoHeader';
+import { storageSetItem, storageGetItem } from '../utils/storage';
+import { IApp, ITodo } from '../types/todoTypes';
+
+const storageTodos = storageGetItem('todos', '');
+let IDX =
```

Comment:
P4: `IDX`가 any 타입으로 되어 있네요! number 타입으로 선언해주면 될 거 같아요.

```suggestion
let IDX: number =
```